Library/Homebrew/cask/installer.rb

```diff
@@ -96,6 +97,10 @@ def install
       check_conflicts
 
       print caveats
+      if dry_run?
+        puts "#{Formatter.identifier(@cask)} would be installed"
```

Comment:
I think this needs handled in the caller instead so that we can output multiple casks in the same format that we do with formulae